ProjectSidewalk/SidewalkWebpage
# 3423 Recording Panorama Dates and Histories from GSV API

Resolves #3423

In this PR, I implemented functionality to record all the panorama histories of each panorama that all users visit, and store them in a new table in the database in case we want to use that data later. I did this by first modifying the frontend to collect and keep track of the visited pano ids & histories given by the GSV API for each explore and validate mission that the user does. I then made a table in the database and wrote some backend code inside the existing explore and validate data submission routes to get that data from the frontend and store it in the new table.  

The schema for the table is the following:
panorama_id (primary key)
visited_timestamp (timestamp)
pano_month (int)
pano_year (int)
location_current_pano_id (foreign key pointing to an existing panorama_id in the same table)

The idea here is that if we want to get all the previous panos at the same location as some pano, we can look for all panos with a location_current_pano_id matching that panorama's location_current_pano_id.

##### Testing instructions
1.  Delete everything from the table using the deleteAll() method. 
2. Write a print statement in the backend route (validation or audit route) that prints out the method call of selectAllPanoramaHistories(). This will print out all the rows in the table.
3. Step through a label or validate mission and observe how the table changes
4. If you'd like to test modifying a row, what I did was write a quick method in the PanoramaHistoryTable.scala that inserts a row with a primary key that's already in the table, and different data for the other columns. Re-print out the entire table and observe that that row is different.

##### Things to check before submitting the PR
